envoy/config/metrics/v2/stats.proto

```diff
@@ -165,9 +165,5 @@ message DogStatsdSink {
     // The UDP address of a running DogStatsD compliant listener. If specified,
     // statistics will be flushed to this address.
     envoy.api.v2.core.Address address = 1;
-
-    // The name of a cluster that is DogStatsD compliant TCP listener. If specified,
-    // Envoy will connect to this cluster to flush statistics.
-    string tcp_cluster_name = 2;
```

Comment:
Thanks. Can you add a comment like https://github.com/envoyproxy/data-plane-api/blob/f8cdf44abbb4bbc66f8633eb8288327b8a1dd151/envoy/api/v2/cds.proto#L44 to this message, so that we skip to field number 3 next? Even if nobody is using this field, we don't want to reuse the field number for wire compatibility reasons.

Reply:
Cool. Is this correct? e6fa3f7

Reply:
You should also add a `reserved` field for number 2, as describe here: https://github.com/envoyproxy/data-plane-api/blame/master/STYLE.md#L26

Reply:
I missed the document, thanks. 36500bf